docs/core/development/policy/coding-standard.rst

```diff
@@ -625,6 +625,126 @@ Any attribute, method argument, or method name that corresponds *directly* to a
 All SQL keywords should be in upper case.
 
 
+Emitting logs
+-------------
+
+All new code emitting log messages should use the new logging system.
+
+This section is dedicated to using the new logging system in Twisted core code.
+For general information about using the logging system please check the dedicated documentation about :doc:`how to log with twisted.logger </core/howto/logger>` and the :api:`twisted.logger.Logger <Logger>` API docs.
+
+General usage rules:
+
+* Don't use explicit namespaces.
+  The logger will automatically fill the namespace.
+* Use the global log publisher.
```

Comment:
Sharing global state is at best an unfortunate compromise.  Given that this is the default, why is this the guidance?  Do we expect users to do something else?